concordium-node/src/stats_export_service.rs

```diff
@@ -352,6 +360,26 @@ impl StatsExportService {
         self.outbound_low_priority_consensus_size.store(value as usize, Ordering::Relaxed);
     }
 
+    /// Gets the timestamp for the last throughput check.
+    pub fn get_last_throughput_measurement(&self) -> i64 {
```

Comment:
Shouldn't this be named with `timestamp` somewher in the string so it is clear that is what it returning.